### DISCUSSION TITLE
add_job() function

### DISCUSSION THREAD

**Discussion by ubuster** [cybertec-postgresql/pg_timetable#288] [Q&A] [answered]
add_job() function is non-functional on my database until I removed the attribute "strict". Is it possible?

**Comment by pashagolub** [comment]
Hello,

what version affected?

**Reply by ubuster** [reply]
Hello Pasha,
pg_timetable_4.0.0-beta.1_Windows_x86_64

**Reply by pashagolub** [reply]
Thanks! Will check! Please stay tuned...

**Reply by pashagolub** [reply]
Would you please paste the SQL caused the issue? 

Thanks in advance!

**Comment by ubuster** [comment]
Well I'll try it.
When I use an example command from README.md - e.g. "SELECT timetable.add_job('execute-func', '5 0 * 8 *', 'SELECT public.my_func()')" - the result is null (nothing). After removing the clause "strict" the result is a bigserial number and tables timetable.chain, timetable.parameter and timetable.task content some filled row(s).

**Reply by pashagolub** [reply]
Thanks. It's really the bug. Thanks for pointing it out! 🙇

**Reply by pashagolub** [reply]
Fixed by #292

**Comment by pashagolub** [comment, marked as answer]
Fixed by #292
